modules/openapi-generator/src/main/resources/python-flask/base_model_.mustache

```diff
@@ -22,34 +22,37 @@ class Model(object):
     attribute_map = {}
 
     @classmethod
-    def from_dict(cls{{^supportPython2}}: typing.Type[T]{{/supportPython2}}, dikt){{^supportPython2}} -> T{{/supportPython2}}:
```

Comment:
Why did you remove this code which works for python 2.7?
We still need this because we support python 2.7 and higher.
Can you add it back in?